### DISCUSSION TITLE
Student Dev Pack

### DISCUSSION THREAD

**Discussion by phreakin** [community/community#58288] [GitHub Education]
### Hi everyone,

My student dev pack expired like everyone else. I have uploaded my id, my current registration info, and everything. Yet this automated system doesn't process anything ever. Honestly, I'm to the point of using something else if it's going to be this absurd. Honestly, since MS took over, the "support" of being referred to here reminds me of the good old MS Premier/Partner support days. Where literally you just went around in a circle until you quit and were frustrated. I just want to use it for school. I'm almost 50 and the clock is ticking for me you know? :)

**Comment by aahouzi** [comment]
If you're experiencing difficulties with the automated system or if your documentation hasn't been processed, it's recommended to reach out to the support channels provided by the service or organization offering the student developer pack. They should be able to assist you with the verification process and address any issues you're facing. Be sure to provide them with all the necessary information and any relevant details about your situation.

While it can be frustrating when things don't go as smoothly as expected, it's important to remain patient and persistent. Keep reaching out to the appropriate support channels and explain your situation clearly. They should be able to assist you and help resolve any issues you're facing.

Remember, there are often alternative resources available for students and developers, so if one option doesn't work out, it's worth exploring other possibilities. Good luck, and I hope you're able to get the support you need to continue your studies and development work!

**Comment by samus-aran** [comment]
Hi @phreakin 👋 

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.
